tests/ttnn/integration_tests/yolox_m/test_ttnn_dark2.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import torch
+
+from ttnn.model_preprocessing import preprocess_model
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from models.utility_functions import skip_for_wormhole_b0
+
+
+from models.experimental.functional_yolox_m.reference.dark2 import Dark2, Focus
+from models.experimental.functional_yolox_m.tt.ttnn_dark2 import TtDark2, TtFocus
+
+import tests.ttnn.integration_tests.yolox_m.custom_preprocessor_d2 as D2
+import ttnn
+
+
+def create_custom_preprocessor(device):
+    def custom_preprocessor(model, name, ttnn_module_args):
+        parameters = {}
+        parameters = D2.custom_preprocessor(device, model, name, ttnn_module_args)
+        return parameters
+
+    return custom_preprocessor
+
+
+import pytest
```

Comment:
Keep imports at one place and check if we are using all the imports. Remove any unnecessary imports.